test/dynamics/signals/test_signals.py

```diff
@@ -873,31 +874,26 @@ def test_DiscreteSignalSum_iterator(self):
         self.assertAllClose(sum_val, self.discrete_sig_sum(3.0))
 
 
-class TestSignalJax(TestSignal, TestJaxBase):
+class TestSignalJax(TestJaxBase, TestSignal):
```

Comment:
Method inherited at first has higher priority. (TestSignalJax use `asarray` in TestJaxBase.)